class.jetpack.php

```diff
@@ -6379,6 +6379,11 @@ public function deprecated_hooks() {
 			'jetpack_enable_site_verification'                       => null,
 			'can_display_jetpack_manage_notice'                      => null,
 			'can_display_jetpack_manage_notice'                      => 'jetpack_json_manage_api_enabled',
+			// Removed in Jetpack 7.3.0
```

Comment:
❤️ a lot easier than I thought!

Reply:
Yup, that at least announces to folks who are using old filters and 5079584 will remove user meta on the next time that user is updated. I abstracted it enough since there's probably future meta we can remove in the future.